serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
+Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a subscription.
+
+The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap, however developers can still create their own channels directly by instantiating a supported channel object.
+
+== Supported channel types
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
+
+[IMPORTANT]
+====
+InMemoryChannel channels are a development only channel and are not suitable for use in production.
+====
+
+The following are limitations of InMemoryChannel channels:
+
+* No event persistence is available. If a pod is goes down, events on that pod will be lost.
+* InMemoryChannel channels do not implement event ordering, so two events that are received in the channel at the same time may be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no redelivery attempts. Instead, the rejected event will be sent to a dead letter sink if this exists, or is otherwise dropped.
```

Comment:
"dead letter sink" - Would it be more apt to call this a "dead letter queue"?
s/if this exists, or is otherwise dropped/if this exists; otherwise, it is dropped./

Reply:
Again, this refers to a specific sink-related thing. I'm going to follow up with another PR when I have time to discuss the dead letter sink properly.